libs/environment/src/components/node-switcher/layout-cell.tsx

```diff
@@ -0,0 +1,26 @@
+import type { ReactNode } from 'react';
+import clsx from 'clsx';
+import { t } from '@vegaprotocol/react-helpers';
+
+type LayoutCellProps = {
+  hasError?: boolean;
+  isLoading?: boolean;
+  children?: ReactNode;
+};
+
+export const LayoutCell = ({
+  hasError,
+  isLoading,
+  children,
+}: LayoutCellProps) => {
+  return (
+    <div
+      className={clsx('px-8 text-right', {
```

Comment:
created [this ticket](https://github.com/vegaprotocol/frontend-monorepo/issues/674) some time ago, because our repo is full of clsx and classnames, would be nice to not have them both, or have some indication which one to use.

Reply:
hmm, interesting, i remember checking it some time ago and i've seen precedents, but not anymore. ticket seems redundant, cleaned up clsx from here, too. thanks for the catch!